hack/validate/shfmt

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+# NOTE: `git grep '^#!'` may also pick up non-shell script files.
+# Add `| grep -v exceptional_file.foo` if any false-positive is detected.
+git grep --name-only '^#!' | grep -v vendor | grep -v '\.go$' | grep -v Jenkinsfile |
+	xargs shfmt -d
```

Comment:
I pushed a branch with an extra commit so see what it would look like; https://github.com/moby/moby/compare/master...thaJeztah:shfmt2_alt?expand=1&w=1

Or (easier to read) with indentation whitespaces ignored;
https://github.com/moby/moby/compare/master...thaJeztah:shfmt2_alt?expand=1&w=1

Reply:
Both URLs seem same 😅

Reply:
oh! my bad; one of them should not have `&w=1`

Reply:
Diff in https://github.com/moby/moby/compare/master...thaJeztah:shfmt2_alt?w=1 LGTM!
(thanks to you both for working on this :heart: :+1:)

Reply:
updated to use the suggested flags, thanks